packages/core/src/components/popover/popover.tsx

```diff
@@ -560,7 +557,7 @@ export class Popover extends AbstractComponent<IPopoverProps, IPopoverState> {
 
     private updateTether() {
         if (this.state.isOpen && !this.props.inline && this.popoverElement != null) {
-            const { constraints, position, tetherOptions, useSmartPositioning } = this.props;
+            const { constraints, position, tetherOptions = {}, useSmartPositioning } = this.props;
```

Comment:
props defaults are usually set with `defaultProps`... why are you changing this?